packages/models-library/src/models_library/api_schemas_webserver/catalog.py

```diff
@@ -100,18 +99,137 @@ class Config(_BaseCommonApiExtension.Config):
 ServiceOutputsGetDict: TypeAlias = dict[ServicePortKey, ServiceOutputGet]
 
 
-_EXAMPLE: dict[str, Any] = deepcopy(
-    api_schemas_catalog_services.ServiceGet.Config.schema_extra["example"]
-)
-_EXAMPLE.update(
-    {
+_EXAMPLE_FILEPICKER: dict[str, Any] = {
+    **api_schemas_catalog_services.ServiceGet.Config.schema_extra["example"],
+    **{
         "inputs": {
             f"input{i}": example
             for i, example in enumerate(ServiceInputGet.Config.schema_extra["examples"])
         },
         "outputs": {"outFile": ServiceOutputGet.Config.schema_extra["example"]},
-    }
-)
+    },
+}
+
+
+_EXAMPLE_SLEEPER: dict[str, Any] = {
+    "name": "sleeper",
+    "thumbnail": None,
+    "description": "A service which awaits for time to pass, two times.",
+    "classifiers": [],
+    "quality": {},
+    "accessRights": {"1": {"execute_access": True, "write_access": False}},
+    "key": "simcore/services/comp/itis/sleeper",
+    "version": "2.2.1",
```

Comment:
we use the sleepers 1.0.0 in a lot of tests, it is a good representative of the very old services.

Reply:
In this context is not relevant.